PurchasesTests/Mocks/MockReceiptFetcher.swift

```diff
@@ -16,8 +23,7 @@ class MockReceiptFetcher: ReceiptFetcher {
         self.init(requestFetcher: requestFetcher, bundle: .main)
     }
 
-    @objc override public func receiptData(refreshPolicy: ReceiptRefreshPolicy,
-                                           completion: @escaping ((Data?) -> Void)) {
+    override public func receiptData(refreshPolicy: ReceiptRefreshPolicy, completion: @escaping ((Data?) -> Void)) {
```

Comment:
this `func` is no longer `public`, so this can be 
```suggestion
    override func receiptData(refreshPolicy: ReceiptRefreshPolicy, completion: @escaping ((Data?) -> Void)) {
```

Reply:
✅